negative charge in command line

**Describe the bug**
Cannot set negative charges in command line (anymore)


 xtb version 6.3.0 (007a1746ce07a5e37ef07acbf67b7b80c5373fa6) compiled by '@Linux' on 05/28/2020

```
xtb coord.xyz -P 16 --uhf 5 --chrg -5 
########################################################################
[ERROR] Program stopped due to fatal error
-4- Command line argument parsing failed
-3- prog_main_parseArguments: Unknown option '-5' provided
-2- prog_main_parseArguments: Molecular charge is not provided
```

--- a/src/prog/argparser.f90
+++ b/src/prog/argparser.f90
@@ -102,6 +102,8 @@ subroutine initArgument(self, iArg)
    integer, intent(in) :: iArg
 
    integer :: length
+   character(len=*), parameter :: flagchars = &
+      'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ_-'
 
    self%unused = .true.
 
@@ -110,7 +112,8 @@ subroutine initArgument(self, iArg)
    call get_command_argument(iArg, self%raw)
 
    inquire(file=self%raw, exist=self%isFile)
-   self%isFlag = index(self%raw, '-') == 1
+   self%isFlag = index(self%raw, '-') == 1 &
+      & .and. (len(self%raw) > 2 .or. verify(self%raw, flagchars) == 0)
 
 end subroutine initArgument
 

